### ISSUE TITLE
Librería itertools

### ISSUE THREAD

**Issue by Arcoirisky** [PUC-IIC2283/2020-IIC2283#9]
¿Podemos usar itertools para la P2 de la tarea? 
Específicamente necesito combinations para probar algo, quizás no lo siga usando después pero porsiacaso

**Comment by PabloMessina** [comment]
Sí, no problem, todo lo que viene en la librería estándar de Python (sin pip installar cosas extras) se puede usar.
